build_dependencies.sh

```diff
@@ -314,8 +326,8 @@ verify_qt()
 build_all()
 {
   ## create build and install directories
-  if [[ -z $BUILD_DIR ]];   then BUILD_DIR=${SRC}/dependencies/build;     fi
-  if [[ -z $INSTALL_DIR ]]; then INSTALL_DIR=${SRC}/dependencies/install; fi
+  if [[ -z $BUILD_DIR ]];   then BUILD_DIR=${SRC}/dependencies3/build;     fi
```

Comment:
Should this be dependencies3 now?

Reply:
Nope I will change that, that was on my local machine.